### PR TITLE
Feat: Cache library upgrade

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Upgrades cache and database libraries, both related to ID case sensitivity issues.

## Test Plan

Unit tests on library-level + current tests must pass.

## Related PRs and Issues

https://github.com/appwrite/appwrite/issues/2959

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,10 +42,10 @@
         "utopia-php/abuse": "0.7.*",
         "utopia-php/analytics": "0.2.*",
         "utopia-php/audit": "0.8.*",
-        "utopia-php/cache": "0.4.*",
+        "utopia-php/cache": "0.5.*",
         "utopia-php/cli": "0.12.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.15.*",
+        "utopia-php/database": "dev-feat-cache-upgrate as 0.15.1",
```
Comment (christyjacob4) [inline composer.json:48]: Reminder to use the correct library version 👍

File: app/init.php [hunk 2]
```diff
@@ -135,6 +135,8 @@
 // Response related
 const MAX_OUTPUT_CHUNK_SIZE = 2*1024*1024; // 2MB
 
+Cache::setCaseSensitivity(false);
```
Comment (christyjacob4) [inline app/init.php:138]: I had a similar comment in here 
https://github.com/utopia-php/database/pull/117

If majority of use cases require the caseSensitivity to be turned off, lets keep false as the default in 
https://github.com/utopia-php/cache/blob/02ffbe165f9632e05519eb75a2ce32deb19638a0/src/Cache/Cache.php#L15

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
PR looks good 👍 
Just one comment

File: composer.json (hunk 1 above)
Comment (Meldiron) [reply composer.json]: Updated

**Review by eldadfux — APPROVED** [review]

**Pull request merged**